.gitignore

```diff
@@ -1,5 +1,5 @@
 addon-*.zip
 _releases
-_data
-hashes.json
+# _data
+# hashes.json
```

Comment:
FYI, but I generally don't commit this into the git repo and just store the _output folder. It does mean I need to build the release artifacts but that is ok as it makes dropping the git repo into a development environment easier.

Reply:
I see. I'll upload the _output folder in a few minutes!